.github/workflows/kind.yml

```diff
@@ -534,6 +534,8 @@ jobs:
     - name: Load Antrea image
       run: |
         docker load -i antrea-ubuntu.tar
+        docker tag antrea/antrea-agent-ubuntu-coverage:latest antrea/antrea-agent-ubuntu:latest
+        docker tag antrea/antrea-controller-ubuntu-coverage:latest antrea/antrea-controller-ubuntu:latest
```

Comment:
have we been using the wrong image the whole time (i.e., pulling the image from the registry instead of using the one we built)?

this should probably be in its own PR, that we can backport

Reply:
I think so and I just found the issue when I verified this change. Will make a separate PR for this.